video/out/w32_common.c

```diff
@@ -953,6 +953,28 @@ static bool is_visible(HWND window)
     return GetWindowLongPtrW(window, GWL_STYLE) & WS_VISIBLE;
 }
 
+static bool update_transparency(struct vo_w32_state *w32, bool handleDefault) {
+    if (!w32 || w32->parent) {
+        return false;
+    }
+
+    int transparency = w32->opts->window_transparency;
+
+    if (transparency == -1 && handleDefault) { // Reset the window attributes
+        // Remove the WS_EX_LAYERED flag from the extended style and set the alpha value to 255
```

Comment:
Is 0-255 always the legal range, even if the monitor's space has more than 8-bit depth?

Reply:
My monitor is set to 10bit and the same range worked in my last testing, I only have 10(8+FRC) though, but I assume code-wise windows would handle that the same no?

Reply:
If you tested it on 10-bit and it worked that sounds correct to me.

Reply:
Might need someone to test on windows 7? I'm not sure, but if we're purely going by API support windows has all the used features since at least 7.